classes.py

```diff
@@ -123,116 +130,356 @@ def from_dict(data):
 
         return person
 
+
+class Show:
+    _id_counter = 0
+
+    def __init__(self, show_id, event_id, start_time, end_time, price, datetime):
+        self.show_id = Show._get_next_id()
+        self.event_id = Event._get_next_id()
```

Comment:
Nie rozumiem czemu tu jest też generowany nowy event.... chodzi o to że jak generujesz show, to też musi być ten pierwszy Event? Ale czy najpierw generujemy Event a potem Show czy od razu pierwszy Show (razem z Eventem)

Reply:
Generujemy Event -> dalej Show

Reply:
No to jak już jest Event, to należy go do Show przywiązać (przekazac jako parametr)